test/multipart.jl

```diff
@@ -21,4 +21,13 @@
             @test_throws MethodError HTTP.post(uri, body)
         end
     end
+    @testset "HTTP.Multipart ensure show() works correctly" begin
+		# testing that there is no error in printing when nothing is set for filename
+		str = sprint(show, (HTTP.Multipart(nothing, IOBuffer("some data"), "plain/text", "", "testname")))
+		@test findfirst("contenttype=\"plain/text\"", str) != nothing
+	end
+	@testset "HTTP.Multipart test constructor" begin
```

Comment:
Just needs to be de-intended (is that a word?)

Reply:
Looks like my editor inserted tabs rather than spaces, fixed the issue, it looks fine now, squashed it into the the main rev.